config/locales/helpers/en.yml

```diff
@@ -4,6 +4,8 @@ en:
     anonymous_byline: Anonymous
     anonymous_with_name_byline_html: Anonymous [%{pseud_byline}]
     archivist_byline_html: "%{external_creator} [archived by %{pseud_byline}]"
+    help_modal:
+      question_mark: "?"
```

Comment:
I was going to suggest `help_symbol` in case we want to change it to some other symbol in the future ("!" and "i" are also common, for example), but considering the class is `question`, we might as well stick with `question_mark`.

Reply:
I prefer `help_symbol`, since it makes clearer that other locales can use whatever they want here, and the classes are also `help symbol`